### DISCUSSION TITLE
Stuck with Deprecated Libraries on .NET Framework 4.8.x

### DISCUSSION THREAD

**Discussion by cobalto** [CoreWCF/CoreWCF#1426] [Q&A]
Due to the current status of our project, we need to migrate our server application to CoreWCF while still on `.Net 4.8.x`. In a later iteration, we will migrate to `.Net 8`. So far, we have been able to migrate all our WCF services to CoreWCF with minimal changes.

However, as we enter the late phase of this stage, we took the time to review all the libraries added during the migration and their versions in use.

We noticed that, by default, the project points to some libraries from AspNetCore that are currently marked as `Deprecated`. We found multiple discussions ([#1147](https://github.com/CoreWCF/CoreWCF/issues/1147), [#603](https://github.com/CoreWCF/CoreWCF/discussions/603)) indicating either to upgrade where possible or downgrade to the `2.1.x` version of the packages.

However, at some point, we got stuck without the possibility to move either up or down.

For example:

**CoreWCF.ConfigurationManager** `1.5.2`
- Requires **CoreWCF.WebHttp** `(>= 1.5.2)`

**CoreWCF.WebHttp** `1.5.2`
- Requires **Microsoft.AspNetCore.WebUtilities** `(>= 2.2.0)`
- This version of **Microsoft.AspNetCore.WebUtilities** is deprecated.

In this case, I cannot downgrade **Microsoft.AspNetCore.WebUtilities** because its version is bound with **CoreWCF.WebHttp**, nor can I upgrade it to a newer version because `2.2.0` was the last version targeting `net48` or `net481` (actually, even `net5`, `net6`, and `net7`). 

The next available version of the package is `8.0.0`, which only targets `net8`.

What is the correct procedure here?
Should we keep the deprecated references or there are any other solution that I missed?

**Comment by g7ed6e** [comment]
My understanding is that Microsoft.AspNetCore.* 2.1.x packages are the latest packages supported on .NET Framework.
@mconnew should we downgrade the dependency on Microsoft.AspNetCore.WebUtilities to 2.1.1  ?
